### PR TITLE
feat: handle airgapped deployment

### PR DESCRIPTION
**Pull request by hectorj2f**
**What type of PR is this?**

Feature
**What this PR does/ why we need it**:

Allow adding certain settings when having an airgapped deployment.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [x] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by vespian — APPROVED** [review]

**Review by drewvanstone — APPROVED** [review]

**Comment by hectorj2f** [comment]
I re-run the tests... ⭕

**Review by armandgrillet — APPROVED** [review]

**Comment by hectorj2f** [comment]
@jimmidyson The federated addons are not all using `https://mesosphere.github.io/charts/stable`. This value does not fit to all, however the value when it is airgapped. It does.

**Comment by jimmidyson** [comment]
Now I look at the PR again, I realised that airgapped should be applied to all or none of the federated addons, correct? If so, then I propose to move the value to `global.federated.airgapped` and use that in the subcharts. This would allow a single setting to apply to all.

File: stable/kommander/Chart.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ maintainers:
   - name: alejandroEsc
   - name: jimmidyson
 name: kommander
-version: 0.10.3
+version: 0.10.4
```
Comment (jimmidyson) [inline stable/kommander/Chart.yaml:10]: Let's bump the minor version to reflect adding the airgapped capability:

```suggestion
version: 0.11.0
```

**Review by jimmidyson — APPROVED** [review]
LGTM but please bump version to 0.11.0.

**Comment by hectorj2f** [comment]
We need to bump yakcl with a bugfix, then we could merge this PR.

**Pull request merged**